functions/src/Android Version/index.ts

```diff
@@ -12,43 +12,52 @@ main.use(bodyParser.json());
 
 export const getAndroidVersion = functions.https.onRequest(main);
 
-app.get('/supportedAndroidVersions', (request, response) => {
-    getSupportedAndroidVersions(response);
+app.post('/supportedAndroidVersions', (request, response) => {
+  getSupportedAndroidVersions(response);
 });
 
 function getSupportedAndroidVersions(res: any) {
```

Comment:
this also just looks like a bunch of formatting changes, hard to tell what if anything is actually being functionally changed. Can we separate out the formatting fixes into its own PR?

Reply:
same for all the other functions

Reply:
Sorry about that I'll fix it